add-on/chrome/bootstrap.js

```diff
@@ -912,12 +912,13 @@ var WindowListener = {
        * Handles events from gBrowser.
        */
       handleEvent: function(event) {
+
         switch (event.type) {
           case "DOMTitleChanged":
             // Get the new title of the shared tab
             this._notifyBrowserSwitch();
             break;
-          case "TabSelect":
+          case "TabSelect": {
```

Comment:
Similarly what changed to trigger `no-case-declarations`? It looks like it's from [`eslint:recommended`](http://eslint.org/docs/rules/#best-practices)